client/src/resources/lang/en_us.json

```diff
@@ -1031,15 +1032,15 @@
     "wiki.article.role.witch.attributes":"* The first player you visit is told that they were possessed or immune to possession\n* You are told all night information messages that the player you possess gets told",
     "wiki.article.role.witch.extra":"* You are immune to getting roleblocked and possession\n* You can't select other mafia members for your first target, but you can select any living player for your second target\n* Your second target gets erased after you change your targets visit, so no other roles can interact with it\n    * A tracker will only see you visit your first target",
 
-    "wiki.article.role.janitor.guide": "- At night, select a player who dies to clean them\n    - Cleaning a player on the night they die erases their role and alibi from their grave so nobody knows who they are\n    - You are told the cleaned players role and alibi\n- You can only use your ability three times\n\nYou should select the same player as the godfather so your ability actually works.",
-    "wiki.article.role.janitor.abilities":"* Visit a player to clean their grave only if they die tonight\n* If a grave is cleaned, that player's role and alibi do not appear in the graveyard\n* You are told the role and alibi of the player you cleaned.",
-    "wiki.article.role.janitor.attributes":"* You only have three uses of your ability",
-    "wiki.article.role.janitor.extra":"* If you visit a player to attempt to clean them, it costs an ability use, even if they don't die\n* If you are roleblocked, it doesn't cost an ability use\n* If you visit a jailed player, it doesn't cost an ability use\n* You can't target other mafia members",
+    "wiki.article.role.mortician.guide": "- At night, select a player to tag them for cremation\n    - A player who is tagged for cremation, gets cremated when they die, erasing their role and alibi from their grave so nobody knows who they are\n    - You are told the cremated players role and alibi when they die\n- You can only tag three players",

```

Comment:
A player who is tagged for creation (no comma) get cremated when they die

Reply:
yep, fixed